docs/janssen-server/scim/config.md

```diff
@@ -46,13 +46,17 @@ These contents should be then passed to the `patch-scim-config` operation, e.g.
 
 To retrieve the current server configuration using TUI proceed as below:
 
-1. Launch TUI, e.g. by running `jans tui`, and follow the prompts
-1. Highlight the SCIM tab using your keyboard's left/right arrow key
-1. Highlight the "Get Scim Configuration" button using the tab key
-1. Press enter
+1. Launch TUI, e.g. by running `python3 /opt/jans/jans-cli/jans_cli_tui.py`, and follow the prompts
```

Comment:
To run TUI, we are now using `jans tui` wrapper command. So the document should use the same.